doc/rst/technotes/libraries.rst

```diff
@@ -260,6 +260,26 @@ An example Makefile which uses the generated ``Makefile.foo`` looks like this:
 
 .. _Makefileless Compilation In Single Locale:
 
+CMake Helper
+~~~~~~~~~~~~
+
+Similar to the makefile helper, the Chapel compiler can also generate a
+CMakeLists file containing the includes directories and linker flags that must
+be added to a CMake project to properly compile. Such a CMakeLists file can be
+generated using ``--library-cmakelists``.
+
+This CMakeLists file defines ``CHPL_INCLUDE_DIRS`` and ``CHPL_LINK_LIBS`` which can
```

Comment:
Didn't you rename these?

Reply:
Yes! Thank you for catching this one (and the other documentation misses)